python/mxnet/base.py

```diff
@@ -120,7 +132,7 @@ def c_array(ctype, values):
 
     Parameters
     ----------
-    ctype : ctypes data type
+    ctype : ctypes data type such as mx_float
```

Comment:
Is this the right use of the formal parameter type specification - @piiswrong @mli can you comment?